db/migrations/versions/6ad072a15e50_.py

```diff
@@ -0,0 +1,88 @@
+"""empty message
+
+Revision ID: 6ad072a15e50
+Revises: 14bff16bccc8
+Create Date: 2023-06-27 16:39:06.293214
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from alembic_utils.pg_extension import PGExtension
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "6ad072a15e50"
+down_revision = "14bff16bccc8"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "assessment_flag",
+        sa.Column("id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column(
+            "application_id", postgresql.UUID(as_uuid=True), nullable=True
+        ),
+        sa.Column(
+            "latest_status",
+            postgresql.ENUM(
+                "RAISED", "STOPPED", "RESOLVED", name="flagstatus"
+            ),
+            nullable=True,
+        ),
+        sa.Column("latest_allocation", sa.String(), nullable=True),
+        sa.Column(
+            "sections_to_flag",
+            postgresql.ARRAY(sa.String(length=256)),
+            nullable=True,
+        ),
+        sa.ForeignKeyConstraint(
+            ["application_id"],
+            ["assessment_records.application_id"],
+            name=op.f("fk_assessment_flag_application_id_assessment_records"),
+        ),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_assessment_flag")),
+    )
+    op.create_table(
+        "flag_update",
+        sa.Column("id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column(
+            "assessment_flag_id", postgresql.UUID(as_uuid=True), nullable=True
+        ),
+        sa.Column("user_id", sa.String(), nullable=False),
+        sa.Column(
+            "date_created",
+            sa.DateTime(),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.Column("justification", sa.String(), nullable=True),
+        sa.Column(
+            "status",
+            postgresql.ENUM(
+                "RAISED", "STOPPED", "RESOLVED", name="flagstatus"
+            ),
+            nullable=True,
+        ),
+        sa.Column("allocation", sa.String(), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["assessment_flag_id"],
+            ["assessment_flag.id"],
+            name=op.f("fk_flag_update_assessment_flag_id_assessment_flag"),
+        ),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_flag_update")),
+    )
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
```

Comment:
getting these errors when I run downgrade command `flask db downgrade 14bff16bccc8`. Can you please check this?
```
INFO  [alembic.runtime.migration] Running downgrade 6ad072a15e50 -> 14bff16bccc8, empty message
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1900, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 736, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.DuplicateObject: extension "pg_trgm" already exists
```

Reply:
Good spot, sorted now 👍